src/input_file.rst

```diff
@@ -1169,23 +1182,39 @@ Note that some tags, or entries for tags, are only used in certain ensembles (e.
     Fugacity  Si     0.0
     Fugacity  O      0.0
 
-  - Value 1: String - The residue to apply this fugacity.
+
+    (MPI-GOMC Only)
+
+  - Value 1: String
```

Comment:
Complete your description: - Value 1: String - The residue name to apply this fugacity.